docs/reference/commandline/swarm_ca.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "swarm ca"
+description: "The swarm ca command description and usage"
+keywords: "swarm, ca"
+---
+
+<!-- This file is maintained within the docker/docker Github
+     repository at https://github.com/docker/docker/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# swarm ca
+
+```markdown
+Usage:	docker swarm ca [OPTIONS]
+
+Manage root CA
+
+Options:
+      --ca-cert pem-file          Path to the PEM-formatted root CA certificate to use for the new cluster
+      --ca-key pem-file           Path to the PEM-formatted root CA key to use for the new cluster
+      --cert-expiry duration      Validity period for node certificates (ns|us|ms|s|m|h) (default 2160h0m0s)
+  -d, --detach                    Exit immediately instead of waiting for the root rotation to converge
+      --external-ca external-ca   Specifications of one or more certificate signing endpoints
+      --help                      Print usage
+  -q, --quiet                     Suppress progress output
+      --rotate                    Rotate the swarm CA - if no certificate or key are provided, new ones will be generated      
+```
+
+## Description
+
+View or rotate the current swarm CA certificate. This command must target a manager node.
+
+## Examples
+
+```bash
+$ docker swarm ca
+  -----BEGIN CERTIFICATE-----
+  MIIBazCCARCgAwIBAgIUJPzo67QC7g8Ebg2ansjkZ8CbmaswCgYIKoZIzj0EAwIw
+  EzERMA8GA1UEAxMIc3dhcm0tY2EwHhcNMTcwNTAzMTcxMDAwWhcNMzcwNDI4MTcx
+  MDAwWjATMREwDwYDVQQDEwhzd2FybS1jYTBZMBMGByqGSM49AgEGCCqGSM49AwEH
+  A0IABKL6/C0sihYEb935wVPRA8MqzPLn3jzou0OJRXHsCLcVExigrMdgmLCC+Va4
+  +sJ+SLVO1eQbvLHH8uuDdF/QOU6jQjBAMA4GA1UdDwEB/wQEAwIBBjAPBgNVHRMB
+  Af8EBTADAQH/MB0GA1UdDgQWBBSfUy5bjUnBAx/B0GkOBKp91XvxzjAKBggqhkjO
+  PQQDAgNJADBGAiEAnbvh0puOS5R/qvy1PMHY1iksYKh2acsGLtL/jAIvO4ACIQCi
+  lIwQqLkJ48SQqCjG1DBTSBsHmMSRT+6mE2My+Z3GKA==
+  -----END CERTIFICATE-----
+  
+ $ docker swarm ca --rotate
+ desired root digest: sha256:919778e05147cac104b83487d3223382ab0cbbc5ba3ea0d1d18b5fa4918d110b
+   rotated TLS certificates:  [=========================>                         ] 1/2 nodes
+   rotated CA certificates:   [>                                                  ] 0/2 nodes
+```
+
+### `--rotate`
+
+This can be used if one of the swarm managers may have been compromised, 
+so that the manager can no longer connect to or be trusted by any node in the 
+cluster.  Alternately, CA rotation can be used to give control of the swarm CA 
+to an external CA, or to take control back from an external CA.
+
+To tell swarm to generate a new CA certificate and key, no additional parameters 
```

Comment:
Hmm, maybe just "The `--rotate` flag does not require any parameters to do a rotation, but you can optionally specify a certificate and key, and those will be used instead of an automatically-generated certificate and key."